spec/integration/type/sshkey_spec.rb

```diff
@@ -1,22 +0,0 @@
-#! /usr/bin/env ruby
```

Comment:
hah.  sorry.  
this is what i meant by "Move the test to provider directory as it's testing provider
functionality."

there was just the single test in the type dir that checked that the sshkey target file was created with the correct mode.